### PR TITLE
Add the flex format check

### PR DESCRIPTION
**Pull request by yhe39**
Change-Id: I4fec950e38c40de53e9b05ea6ca6713b39f5dbf7
Jira:
Tests: Camera woeks well, no regrssion
Signed-off-by: HeYue <yue.he@intel.com>

### CODE REVIEW

**Comment by yhe39** [comment]
hi @js0701 and @YuanjunHuang , submit this patch to check the flex format to fix the camera doesn't work issue when update to mapper 2.1. Please help to review.
I will close the camera wa: https://github.com/intel/minigbm/pull/84. Thanks!

File: cros_gralloc/cros_gralloc_helpers.cc [hunk 1]
```diff
@@ -106,3 +106,15 @@ void cros_gralloc_log(const char *prefix, const char *file, int line, const char
 	__android_log_vprint(ANDROID_LOG_ERROR, buf, format, args);
 	va_end(args);
 }
+
+bool is_flex_format(uint32_t format)
+{
+	switch (format) {
+	case DRM_FORMAT_FLEX_IMPLEMENTATION_DEFINED:
```
Comment (YuanjunHuang) [inline cros_gralloc/cros_gralloc_helpers.cc]: are all flex formats listed here already?

Comment (yhe39) [reply cros_gralloc/cros_gralloc_helpers.cc]: I think so, i grep the FLEX in minigbm folder, only find the DRM_FORMAT_FLEX_IMPLEMENTATION_DEFINED and DRM_FORMAT_FLEX_YCbCr_420_888. I also check the code, the two flex format are deemed as NV12 in certain situation.
Thanks!

**Comment by yhe39** [comment]
close as merged. Thanks!

**Pull request closed without merging**